source/_integrations/statistics.markdown

```diff
@@ -155,3 +155,8 @@ unique_id:
   required: false
   type: string
 {% endconfiguration %}
+
+| :exclamation: An important note of `max_age` and `sampling_size`t   |
```

Comment:
This is not how we write notices, please consult our developer documentation for more information on how to write documentation.

Considering the limited message below, I'm not sure if any emphasis is needed at all.